src/Components/BooksList.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import Form from './Form';
+import Book from './Book';
+import { displayBooks } from '../redux/books/books';
+
+const BooksList = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(displayBooks())
+  },[dispatch])
+  dispatch(displayBooks());
```

Comment:
On the console the are infinite re-renders of the `BookList` component because of the `useEffect` dependency issue. This is causing performance issues on your application.
To resolve this.

- Kindly remove line 12 with the code repetition `dispatch(displayBooks());` since this code is already being run inside the hook.
- Kindly remove `dispatch` in the dependency array of the `useEffect` hook. The array must be empty so that the hook only runs once to prevent multiple re-renders.